apis/workloads/v1alpha1/console_authenticator_webhook.go

```diff
@@ -51,5 +54,13 @@ func (c *ConsoleAuthenticatorWebhook) Handle(ctx context.Context, req admission.
 
 	logger.Info(fmt.Sprintf("authentication successful for user %s", user), "event", "authentication.success", "user", user)
 
+	// This endpoint is only called when we are creating console
+	// objects. If we are succeeding the request we are creating a
+	// console request.
+	err = c.lifecycleRecorder.ConsoleRequest(ctx, csl)
+	if err != nil {
+		logging.WithNoRecord(logger).Error(err, "failed to record event", "event", "console.request")
```

Comment:
As an addition later on, it would be cool to increment a Prometheus metric when we hit this condition, so that we can alert on it.

Reply:
We should. I think this would be a nice discussion to have more generally throught the theatre codebase as well.